README.md

```diff
@@ -1,7 +1,6 @@
 # ipns <!-- omit in toc -->
 
-[![ipfs.io](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io)
-[![IRC](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![ipfs.tech](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](https://ipfs.tech)
```

Comment:
Instead of making this change here, can you please PR [these lines](https://github.com/ipfs/aegir/blob/master/src/check-project/readme/header.js#L13-L14) then running `npx aegir check-project` in the root of this repo will update the badges, etc.

Reply:
Done: https://github.com/ipfs/aegir/pull/1073